contrib/openvpn/files/openvpn-server

```diff
@@ -0,0 +1,7 @@
+# openvpn server service
+
+type = process
+command = /usr/bin/openvpn --status /var/log/openvpn-server-status.log --status-version 2 --suppress-timestamps --log /var/log/openvpn-server-service.log --config ${OPENVPN_SERVER:-default}.conf
+working-dir = /etc/openvpn/server
```

Comment:
having a working-dir in /etc seems weird, is this actually required or just a convenience for --config to be relative?

if the latter then pass full path